### PR TITLE
fix(web-components): remove determinate from spinner implementation

### PR DESCRIPTION
**Pull request by chrisdholt**
## Previous Behavior
Spinner optionally could be set as a determinate progress indicator.

## New Behavior
There is no construct for a determinate spinner, the behavior has been removed.

## Related Issue(s)

- Fixes #

### CODE REVIEW

File: packages/web-components/src/spinner/spinner.styles.ts [hunk 1]
```diff
@@ -51,21 +51,7 @@ export const styles = css`
...
-    transform: rotate(-90deg);
-    transition: all 0.2s ease-in-out;
-  }
-
-  :host([appearance='inverted']) .determinite {
-    stroke: ${colorNeutralStrokeOnBrand2};
-  }
-
-  .indeterminate-indicator-1 {
+  .indeterminate-indicator {
```
Comment (spmonahan) [inline packages/web-components/src/spinner/spinner.styles.ts:54]: super nit pick:

```suggestion
  .indicator {
```

File: packages/web-components/src/spinner/spinner.template.ts [hunk 2]
```diff
@@ -1,56 +1,17 @@
...
-  `,
-});
+export const template = html<Spinner>`
+  <template
+    role="progressbar"
+    aria-valuenow="${x => x.value}"
+    aria-valuemin="${x => x.min}"
+    aria-valuemax="${x => x.max}"
+  >
+    <slot name="indeterminate"
```
Comment (spmonahan) [inline packages/web-components/src/spinner/spinner.template.ts:11]:
```suggestion
    <slot name="indicator"
```

File: packages/web-components/src/spinner/spinner.template.ts [hunk 3]
```diff
@@ -1,56 +1,17 @@
...
+      ><svg class="progress" part="progress" viewBox="0 0 16 16">
+        <circle class="background" cx="8px" cy="8px" r="7px"></circle>
+        <circle class="indeterminate-indicator" cx="8px" cy="8px" r="7px"></circle></svg
```
Comment (spmonahan) [inline packages/web-components/src/spinner/spinner.template.ts:14]:
```suggestion
        <circle class="indicator" cx="8px" cy="8px" r="7px"></circle></svg
```

File: packages/web-components/src/spinner/spinner.styles.ts [hunk 4]
```diff
@@ -76,7 +62,7 @@ export const styles = css`
     animation: spin-infinite 3s cubic-bezier(0.53, 0.21, 0.29, 0.67) infinite;
   }
 
-  :host([appearance='inverted']) .indeterminate-indicator-1 {
+  :host([appearance='inverted']) .indeterminate-indicator {
```
Comment (spmonahan) [inline packages/web-components/src/spinner/spinner.styles.ts:65]:
```suggestion
  :host([appearance='inverted']) .indicator {
```

**Review by spmonahan — APPROVED** [review]

File: packages/web-components/src/spinner/spinner.styles.ts (hunk 4 above)
Comment (chrisdholt) [reply packages/web-components/src/spinner/spinner.styles.ts]: Great callout, I love this

**Comment by fabricteam** [comment]
## 📊 Bundle size report

✅ No changes found
<div id="bundleSizeComment9423"/>

File: change/@fluentui-web-components-82d13efa-8e97-478a-b1d9-d6b7c6f32868.json [hunk 5]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (fabricteam) [inline change/@fluentui-web-components-82d13efa-8e97-478a-b1d9-d6b7c6f32868.json:1]: ## [🕵 fluentui-web-components-v3 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31404&commitId=1c232fb413aad443d95ab33f4935f8524be08b43&env=prod&clientType=FLUENTUI)


  <details><summary><b>Accordion</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Accordion.Size - RTL.normal.chromium.png|2| Changed|
|Accordion.Size - Dark Mode.normal.chromium.png|2| Changed|
|Accordion.Size.normal.chromium.png|2| Changed|

</details>


  <details><summary><b>Avatar</b> 5 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar.Active Appearance - Dark Mode.normal.chromium.png|2| Changed|
|Avatar.Active.normal.chromium.png|5| Changed|
|Avatar.Active Appearance.normal.chromium.png|10| Changed|
|Avatar.Color.normal.chromium.png|1| Changed|
|Avatar.Color - RTL.normal.chromium.png|1| Changed|

</details>


  <details><summary><b>Badge</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Badge.Default.normal.chromium.png|1| Changed|
|Badge.Shape.normal.chromium.png|1| Changed|
|Badge.Appearance.normal.chromium.png|1| Changed|
|Badge.Appearance - Dark Mode.normal.chromium.png|15| Changed|
|Badge.Color.normal.chromium.png|1| Changed|
|Badge.Size.normal.chromium.png|6| Changed|

</details>


  <details><summary><b>Button</b> 15 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button.Large Icon Only - Dark Mode.default.chromium.png|4| Changed|
|Button.Large Icon Only.default.chromium.png|5| Changed|
|Button.Large Icon Only - Dark Mode.pressed.chromium.png|5| Changed|
|Button.Large Icon Only.pressed.chromium.png|3| Changed|
|Button.Large Icon Only - Dark Mode.hover.chromium.png|3| Changed|
|Button.Large With Icon - RTL.pressed.chromium.png|3| Changed|
|Button.Large With Icon - RTL.hover.chromium.png|4| Changed|
|Button.Large With Icon.pressed.chromium.png|3| Changed|
|Button.Large With Icon - RTL.default.chromium.png|5| Changed|
|Button.Large With Icon - Dark Mode.pressed.chromium.png|5| Changed|
|Button.Large With Icon - Dark Mode.hover.chromium.png|3| Changed|
|Button.Large Icon Only.hover.chromium.png|4| Changed|
|Button.Large With Icon - Dark Mode.default.chromium.png|4| Changed|
|Button.Large With Icon.default.chromium.png|5| Changed|
|Button.Large With Icon.hover.chromium.png|4| Changed|

</details>


  <details><summary><b>Divider</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Divider.With Svg - RTL.normal.chromium.png|3| Changed|
|Divider.With Svg.normal.chromium.png|3| Changed|
|Divider.Vertical With Svg.normal.chromium.png|4| Changed|

</details>


  <details><summary><b>MenuList</b> 16 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|MenuList.Checkbox With Icons.2nd selected.chromium.png|5| Changed|
|MenuList.Checkbox.2nd selected.chromium.png|1| Changed|
|MenuList.Checkbox With Icons - RTL.2nd selected.chromium.png|5| Changed|
|MenuList.Checkbox With Icons.normal.chromium.png|4| Changed|
|MenuList.Radio With Icons.2nd selected.chromium.png|4| Changed|
|MenuList.Checkbox - RTL.2nd selected.chromium.png|1| Changed|
|MenuList.Radio With Icons.1st selected.chromium.png|4| Changed|
|MenuList.Checkbox With Icons - RTL.1st selected.chromium.png|4| Changed|
|MenuList.Radio With Icons.normal.chromium.png|4| Changed|
|MenuList.With Icons.normal.chromium.png|4| Changed|
|MenuList.Checkbox With Icons.1st selected.chromium.png|4| Changed|
|MenuList.Checkbox With Icons - RTL.normal.chromium.png|4| Changed|
|MenuList.Radio With Icons - RTL.2nd selected.chromium.png|4| Changed|
|MenuList.Radio With Icons - RTL.1st selected.chromium.png|4| Changed|
|MenuList.With Icons.hover menuitem.chromium.png|3| Changed|
|MenuList.Radio With Icons - RTL.normal.chromium.png|4| Changed|

</details>


  <details><summary><b>Slider</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Slider.Size Small.normal.chromium.png|1| Changed|
|Slider.Size Small.rightArrow.chromium.png|1| Changed|

</details>


  <details><summary><b>Switch</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Switch.Checked.hover.chromium.png|1| Changed|
|Switch.Checked - RTL.hover.chromium.png|1| Changed|

</details>


  <details><summary><b>Text</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Text.Block.normal.chromium.png|995| Changed|

</details>


  <details><summary><b>TextInput</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|TextInput.Size Large.normal.chromium.png|2| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>

Comment (chrisdholt) [reply change/@fluentui-web-components-82d13efa-8e97-478a-b1d9-d6b7c6f32868.json:1]: Looks to be noise - all unrelated

**Pull request merged**